### DISCUSSION TITLE
No expression italic since 0.12.0

### DISCUSSION THREAD

**Discussion by mym0404** [KaTeX/KaTeX#2939] [Q&A] [answered]
I am using KaTex 0.11.1 for rendering tex and saving as a HTML rendered result.
`0.11.1`
![0 11 1](https://user-images.githubusercontent.com/33388801/115138658-6cf57280-a068-11eb-86ec-f7ec56bd79fc.jpg)
`0.12.0`
![0 12 0](https://user-images.githubusercontent.com/33388801/115138659-6ebf3600-a068-11eb-9028-a5b7d20d1a40.jpg)

I am newbie at tex syntax and want to know why math characters are non-italic since 0.12.0.
Is there any render option or way to revert this feature with upgrade to upper 0.12.x ?

Thank you.

**Comment by edemaine** [comment, marked as answer]
This sounds like you upgraded the JavaScript code but didn't upgrade the CSS.  Could you check that you're using the matching CSS version?

**Reply by mym0404** [reply]
Thank you for your reply.  😀
In my project, the situation had been reversed.
I stored rendered HTML result(with `0.11.1`) but tried to render with css of `0.13.2`.
In our React Native mobile service, we stored pre-rendered HTML for skipping rendering phase.

**Comment by ylemkimon** [comment]
The class name for the default math font has changed from `mathit` or `mathdefault` to `mathnormal`. Note that it's recommended to use the stylesheet of which version is the same as the one used to render the expression.
